src/components/navigation/Navbar.jsx

```diff
@@ -57,4 +58,36 @@ function Navbar() {
   );
 }
 
-export default Navbar;
+export const TopNavBar = () => {
+  const { user } = UserAuth();
+  const handleSignOut = UseHandleSignOut();
+
+  const NabBarItem = ({ route }) => {
+    return (
+      <li>
+        <NavLink to={route.path}>{route.name}</NavLink>
+      </li>
+    )
+  }
+
+  return (
+    <nav>
+      <ul className="navbar">
+        {routes.map((r, i) => {
```

Comment:
This will iterate through the routes array that is setup and based on the properties, will either show it in the top nav bar or hide it.
This also takes into consideration if the user is authenticated or not and will show the relevant links based on that condition

Reply:
Too smart for me